shared/templates/accounts_password/bash.template

```diff
@@ -3,7 +3,6 @@
 # strategy = restrict
 # complexity = low
 # disruption = low
-. /usr/share/scap-security-guide/remediation_functions
 {{{ bash_instantiate_variables("var_password_pam_" + VARIABLE) }}}
 
-replace_or_append '/etc/security/pwquality.conf' '^{{{ VARIABLE }}}' $var_password_pam_{{{ VARIABLE }}} '@CCENUM@' '%s = %s'
+{{{ bash_replace_or_append('/etc/security/pwquality.conf', '^{{{ VARIABLE }}}', "$var_password_pam_{{{ VARIABLE }}}", '@CCENUM@', '%s = %s') }}}
```

Comment:
I can see too much braces here. You are nesting them, that won't work. 

This file is a template. There are multiple rules that are using this template, they all have same bash remediation which differ only by variable `VARIABLE`  (I know, it's confusing :scream: ).

Check out the the "Templating" section in the documentation: https://complianceascode.readthedocs.io/en/latest/manual/developer/06_contributing_with_content.html#templating

For example, check `linux_os/guide/system/accounts/accounts-pam/password_quality/password_quality_pwquality/accounts_password_pam_dcredit/rule.yml`, then build the `rhel8` product, and then watch what got generated into `build/rhel8/fixes/bash/accounts_password_pam_dcredit.sh`. Currently, you have it broken.

You probably would have to do something similar as is already done on line 6, ie. concatenate the strings with the `VARIABLE` variable.